lib/errors.ts

```diff
@@ -64,6 +64,24 @@ export class ResinMalformedToken extends ResinError {
 ResinMalformedToken.prototype.code = 'ResinMalformedToken';
 
 /**
+ * @summary 
+ * @class
+ * @public
+ *
+ * @param {String} token - token
+ * @return {Error} error instance
+ *
+ * @example
+ * throw new errors.SupervisorLockedError()
+ */
+export class SupervisorLockedError extends ResinError {
```

Comment:
Can you name this class with `Resin` prefix to match the other errors, please?
`ResinSupervisorLockedError`

Reply:
So do I need to call ResinSupervisorLockedError also from the sdk where using this error?

Reply:
Yep, We'll use `ResinSupervisorLockedError` instead of `SupervisorLockedError`.